content/chainguard/chainguard-images/working-with-images/compiled-programs.md

```diff
@@ -0,0 +1,79 @@
+`ca-certificates-bundle`, `chainguard-baselayout`, `glibc`, `glibc-locale-posix`, `ld-linux`, `libcrypto3`, `libgcc`, `libssl3`, `libstdc++`, `openssl`, `openssl-provider-legacy`, and `wolfi-baselayout`.
+
+### wolfi-base
+The `wolfi-base` image is a minimal container image based on the [Wolfi *un-distro*](https://github.com/wolfi-dev/), a community-oriented Linux distribution created by Chainguard to facilitate image builds. The `wolfi-base` image contains `busybox` and `apk-tools` so that you can build your own packages for a custom image. It supports `glibc`.
+
+The following packages are included in the `wolfi-base:latest` Chainguard Image:
+`apk-tools`, `busybox`, `ca-certificates-bundle`, `chainguard-baselayout`, `glibc`, `glibc-locale-posix`, `ld-linux`, `libcrypt1`, `libcrypto3`, `libssl3`, `libxcrypt`, `wolfi-base`, `wolfi-baselayout`, `wolfi-keys`, and `zlib`.
+
+### chainguard-base
+*Paid Image* \
+In addition to the functionality of the `wolfi-base` image, `chainguard-base` reports as being a Chainguard Image, which scanners use to determine what security feeds to reference for vulnerabilities. Additionally, the `chainguard-base` image provides access to vulnerability remediation SLAs to ensure your containers are always up-to-date with the latest releases and patches.
+
+The following packages are included in the `chainguard-base:latest` Chainguard Image:
+`apk-tools`, `busybox`, `ca-certificates-bundle`, `chainguard-baselayout`, `glibc`, `glibc-locale-posix`, `ld-linux`, `libcrypt1`, `libcrypto3`, `libssl3`, `libxcrypt`, `wolfi-base`, `wolfi-baselayout`, `wolfi-keys`, and `zlib`.
+
+
+## What about musl?
+At the time of this writing, no Chainguard Images come packaged with `musl`. Chainguard builds `glibc`-based images because `glibc` is commonly used, which makes it easier for most developers to start consuming Chainguard Images in their environments. Additionally, `glibc` is widely tested, making it a dependable choice for a C standard library implementation. As `glibc` is a well-established option, choosing to use `glibc` ensures more applications will be compatible with new images.
+
+Though `musl` is sometimes chosen because of its minimal footprint, Chainguard’s distroless approach based on [Wolfi](https://www.chainguard.dev/unchained/introducing-wolfi-the-first-linux-un-distro) often results in a container image of comparable (or smaller) size than official `musl` based images. 
+
+## Learn More
+In this article, you learned about the differences between select Chainguard Images, allowing you to make informed decisions about what images to choose for your compiled applications. You can check out our complete suite of Chainguard Images at the [Chainguard Registry](https://images.chainguard.dev/). To learn more about using Chainguard Images, head to the [Chainguard Academy](/chainguard/chainguard-images/), where you can find documentation to help you start incorporating them into your workflow.
```

Comment:
If it were me, I might call this section "next steps" and remove the summary language ("in this article..."). But that might be a matter of preference, so feel free to ignore.